src/main/java/uk/gov/justice/hmpps/prison/service/MovementUpdateService.java

```diff
@@ -66,10 +66,10 @@ public OffenderBooking moveToCell(final Long bookingId, final String internalLoc
     @VerifyBookingAccess
     @HasWriteScope
     public OffenderBooking moveToCellSwap(final Long bookingId, final String reasonCode, final LocalDateTime dateTime) {
-        if(reasonCode != null) checkReasonCode(reasonCode);
-        checkDate(dateTime);
```

Comment:
did you mean to remove the `checkDate` here?

Reply:
I did, it's covered by validateMoveToCell function.